tools/export/uvision4.py

```diff
@@ -28,7 +28,7 @@ class Uvision4(Exporter):
     Exporter class for uvision. This class uses project generator.
     """
     # These 2 are currently for exporters backward compatiblity
-    NAME = 'uVision4'
+    NAME = 'uvision'
```

Comment:
None? all targets were switched to uvision4 progen. Or I might have misunderstood the question?

@theotherjimmy How's NAME used? why does it rename to uvision. I think uvision4 is a better name,  as there's now uvision5. To distinguish. Or did it cause any trouble?